dpypx/autodraw.py

```diff
@@ -23,18 +23,15 @@ def load_image(
             cls, client: Client, xy: tuple[int, int],
             image: Image.Image, scale: float = 1) -> AutoDrawer:
         """Draw from the pixels of an image."""
-        if image.mode == 'RGBA':
-            new_image = Image.new('RGB', image.size)
-            new_image.paste(image, mask=image)
-            image = new_image
+        image = image.convert(mode="RGBA")
         width = round(image.width * scale)
         height = round(image.height * scale)
         # Since the image is only loaded once we can afford to use a high
         # quality filter.
         resized = image.resize((width, height), Image.LANCZOS)
         data = list(resized.getdata())
         grid = [
-            [Pixel(*pixel) for pixel in data[start:start + width]]
+            [(Pixel(*pixel[:3]), pixel[3] > 127) for pixel in data[start:start + width]]
```

Comment:
This line is 88 lines long but the repo tries to keep lines under 80 (I mentioned this when making the suggestion)

Reply:
```suggestion
            [
                (Pixel(*pixel[:3]), pixel[3] > 127)
                for pixel in data[start:start + width]
            ]
```
Line length